wasmtime.hh

```diff
@@ -0,0 +1,2660 @@
+/**
+ * \mainpage
+ *
+ * This project is a C++ API for
+ * [Wasmtime](https://github.com/bytecodealliance/wasmtime). Support for the
+ * C++ API is exclusively built on the [C API of
```

Comment:
Maybe better to do this as part of install-dependencies.sh and install it in /usr/local/include (I guess the library should be in /usr/local/lib64 too).

Reply:
sure, I'll do that